iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -8,6 +8,11 @@
 
 import SwiftUI
 
+protocol LoginCoordinator {
+
+    func showSurvey()
```

Comment:
I think you should consider to name from `surveySelection` or `survey` here  to `Home` as that would be a more common flow that we go from Login screen to Home screen? What do you think? 🤔
```suggestion
    func showHome()
```

Reply:
Updated in https://github.com/blyscuit/kmm-swiftUI-IC/pull/57/commits/de04d90f5ac9e5bbc3b76ee7f3127b8ae887468e